src/PrestaShopBundle/Form/Admin/Sell/CustomerService/MerchandiseReturn/MerchandiseReturnOptionsType.php

```diff
@@ -26,36 +26,74 @@
 
 namespace PrestaShopBundle\Form\Admin\Sell\CustomerService\MerchandiseReturn;
 
+use PrestaShopBundle\Form\Admin\Type\MultistoreConfigurationType;
 use PrestaShopBundle\Form\Admin\Type\SwitchType;
 use PrestaShopBundle\Form\Admin\Type\TranslatableType;
-use Symfony\Component\Form\AbstractType;
+use PrestaShopBundle\Form\Admin\Type\TranslatorAwareType;
 use Symfony\Component\Form\Extension\Core\Type\IntegerType;
 use Symfony\Component\Form\FormBuilderInterface;
 use Symfony\Component\Validator\Constraints\GreaterThanOrEqual;
 
 /**
  * Form type for merchandise returns options
  */
-class MerchandiseReturnOptionsType extends AbstractType
+class MerchandiseReturnOptionsType extends TranslatorAwareType
 {
+    public const FIELD_ENABLE_ORDER_RETURN = 'enable_order_return';
+    public const FIELD_ORDER_RETURN_PERIOD_IN_DAYS = 'order_return_period_in_days';
+    public const FIELD_ORDER_RETURN_PREFIX = 'order_return_prefix';
+
     /**
      * {@inheritdoc}
      */
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
         $builder
-            ->add('enable_order_return', SwitchType::class, [
+            ->add(static::FIELD_ENABLE_ORDER_RETURN, SwitchType::class, [
+                'label' => $this->trans(
+                    'Enable returns',
+                    'Admin.Orderscustomers.Feature'
+                ),
```

Comment:
I believe you forgot the help message here `Would you like to allow merchandise returns in your shop?` that was initially integrated in the form template

Reply:
Ah no it was asked by @Julievrz my bad